### PR TITLE
[fir] Add SPDX license info in metadata 

### PR DESCRIPTION
**Pull request by ansar-rasool**
JIRA-ID: SB-17889

Signed-off-by: ansar-rasool <ansar_rasool@mentor.com>

### CODE REVIEW

**Comment by ansar-rasool** [comment]
List of unchanged files:

README.md
qt5-layer/recipes-qt/qt5/qtbase/0001-add_tracepoint_layer.patch
qt5-layer/recipes-qt/qt5/qtdeclarative/0001-add_tracepoint_layer.patch
recipes-multimedia/gstreamer/gstreamer1.0/0001-Add-optional-LTTng-support-in-configure.patch
recipes-multimedia/gstreamer/gstreamer1.0/0001-gstreamer1.0-meson-add-lttng-tracepoints-support.patch
recipes-multimedia/gstreamer/gstreamer1.0/0002-Add-basic-LTTng-tracepoints-infrastructure.patch
recipes-multimedia/gstreamer/gstreamer1.0/0003-Add-gst_flow-LLTng-tracepoints.patch
recipes-multimedia/gstreamer/gstreamer1.0/0004-gst_flow_data-LLTng-tracepoints.patch
recipes-multimedia/gstreamer/gstreamer1.0/0005-Support-for-gst_flow_event-LLTng-tracepoints.patch
recipes-multimedia/gstreamer/gstreamer1.0/0006-Added-support-for-gst_flow_event_latency-and-gst_flo.patch
recipes-multimedia/gstreamer/gstreamer1.0/0007-Added-GST_TRACEPOINT_ENABLED-macro.patch
recipes-multimedia/gstreamer/gstreamer1.0/0008-Added-support-for-gst_flow_event_caps-LTTng-tracepoi.patch
recipes-multimedia/gstreamer/gstreamer1.0/0009-Added-event-type-field-to-gst_flow_event_qos-event-r.patch
recipes-multimedia/gstreamer/gstreamer1.0/0010-Added-gst_latency_measurement_start-end-passed-event.patch
recipes-multimedia/gstreamer/gstreamer1.0/0011-Added-latency-probe-tracepoint-event.patch

File: recipes-multimedia/gstreamer/gstreamer1.0_1.16%.bbappend [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: LGPL-2.0+
```
Comment (abelal) [inline recipes-multimedia/gstreamer/gstreamer1.0_1.16%.bbappend:2]: this is deprecated. please have a look at https://spdx.org/licenses/ we need to use LGPL-2.0-or-later

File: recipes-multimedia/gstreamer/gstreamer1.0_1.16.3.bbappend [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: LGPL-2.0+
```
Comment (abelal) [inline recipes-multimedia/gstreamer/gstreamer1.0_1.16.3.bbappend:2]: deprecated identifier

File: qt5-layer/recipes-qt/qt5/qtwayland_git.bbappend [hunk 3]
```diff
@@ -1,2 +1,6 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: GFDL-1.3 AND BSD AND ( GPL-3.0 AND The-Qt-Company-GPL-Exception-1.0 OR The-Qt-Company-Commercial ) AND ( GPL-2.0+ OR LGPL-3.0 OR The-Qt-Company-Commercial )
```
Comment (abelal) [inline qt5-layer/recipes-qt/qt5/qtwayland_git.bbappend:2]: GPL-2.0+ is deprecated

**Review by abelal — CHANGES_REQUESTED** [review]
None of the currently accepted identifiers mentioned on the spdx.org use the '+' sign. @ansar-rasool please adjust.

**Comment by abelal** [comment]
Please note that I've mentioned only three instances using + in the review comments but there are others. Please accommodate.

**Review by abelal — APPROVED** [review]

**Pull request merged**